Gulpfile.ts

```diff
@@ -27,7 +27,7 @@ declare global {
 import del = require("del");
 import mkdirP = require("mkdirP");
 import minimist = require("minimist");
-import browserify = require("browserify");
+import browserify = require("browserify")

```

Comment:
We are linting the gulpfile if you do gulp lint. I should update the
jakefile to lint it, too.

On Wed, Jun 22, 2016, 6:42 PM Ron Buckton <notifications@github.com> wrote:

> In Gulpfile.ts
> <https://github.com/Microsoft/TypeScript/pull/9068#discussion_r68164578>:
>
> > @@ -27,7 +27,7 @@ declare global {
> >  import del = require("del");
> >  import mkdirP = require("mkdirP");
> >  import minimist = require("minimist");
> > -import browserify = require("browserify");
> > +import browserify = require("browserify")
>
> nit: trailing semicolon.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Microsoft/TypeScript/pull/9068/files/8aeb682206d4eefffadd2b7de6a614a2cb88c1a6..d008da5888181a6f15823b876c7daa391049c16e#r68164578>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ACzAMuFsOQZ36AA_aGh8JPbvdd9tZITmks5qOeShgaJpZM4Iyq8C>
> .
>